extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1491,13 +1531,17 @@ static public function onArticleSaveComplete( &$article, &$user, $text, $summary
 	}
 
 	/**
-	 * @param $editor
+	 * Don't send email for Wall changes
+	 * @param User $editor
 	 * @param Title $title
 	 * @return bool
 	 */
-	static public function onAllowNotifyOnPageChange( $editor, $title ) {
-		$app = F::app();
-		if ( in_array( MWNamespace::getSubject( $title->getNamespace() ), $app->wg->WallNS ) || $title->getNamespace() == NS_USER_WALL_MESSAGE_GREETING ) {
+	static public function onAllowNotifyOnPageChange( User $editor, Title $title ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$editor". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)